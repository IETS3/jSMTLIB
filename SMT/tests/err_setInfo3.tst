; allowed to set pre-defined info
(set-info :all-statistics x)
;(get-info :all-statistics) ;; non-deterministic
(set-info :reason-unknown x)
(get-info :reason-unknown)
