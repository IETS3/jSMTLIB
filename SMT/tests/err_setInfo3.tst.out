(set-info :all-statistics x)
          ^^^^^^^^^^^^^^^
(error "Setting the value of a pre-defined keyword is not permitted: :all-statistics")
(set-info :reason-unknown x)
          ^^^^^^^^^^^^^^^
(error "Setting the value of a pre-defined keyword is not permitted: :reason-unknown")
unsupported
